src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -1034,6 +1035,37 @@ void testMergeDrsObjectsWithConflictingPath() {
     assertThrows(InvalidDrsObjectException.class, () -> drsService.mergeDRSObjects(drsObjects));
   }
 
+  @Test
+  void testMergeDrsObjectsWithMultipleDescriptions() {
+    DRSObject drsObject1 =
+        createFileDrsObject(
+                "v2_file1",
+                "/my/path/file1.txt",
+                123L,
+                "foomd5",
+                CloudPlatform.GCP,
+                GoogleRegion.ASIA_SOUTH1,
+                Instant.parse("2022-01-01T00:00:00.00Z"))
+            .description("description1");
+    DRSObject drsObject2 =
+        createFileDrsObject(
+                "v2_file1",
+                "/my/path/file1.txt",
+                123L,
+                "foomd5",
+                CloudPlatform.GCP,
+                GoogleRegion.US_CENTRAL1,
+                Instant.parse("2022-01-02T00:00:00.00Z"))
+            .description("description2");
+    List<DRSObject> drsObjects = List.of(drsObject1, drsObject2);
+    assertDoesNotThrow(
+        () -> {
+          DRSObject drsObject = drsService.mergeDRSObjects(drsObjects);
+          assertThat(drsObject.getDescription(), containsString(drsObject1.getDescription()));
+          assertThat(drsObject.getDescription(), containsString(drsObject2.getDescription()));
+        });
```

Comment:
Since an API that throws will fail a text, `assertDoesNotThrow()` isn't needed here.
```suggestion
      DRSObject drsObject = drsService.mergeDRSObjects(drsObjects);
      assertThat(drsObject.getDescription(), containsString(drsObject1.getDescription()));
      assertThat(drsObject.getDescription(), containsString(drsObject2.getDescription()));
```

Note that if you can combine `assertThat()`s on the same item into a single assert:
```
    assertThat(
        drsObject.getDescription(),
        allOf(
            containsString(drsObject1.getDescription()),
            containsString(drsObject2.getDescription())));
```